pkgs/development/tools/cosmocc/default.nix

```
@@ -10,7 +10,7 @@ let
         meta = with lib; {
           homepage = "https://justine.lol/cosmopolitan/";
           description = "compilers for Cosmopolitan C/C++ programs";
-          license = licenses.mit;
+          license = licenses.isc;
```

```suggestion
          inherit (cosmopolitan.meta) license;
```

Also, I suggest 
- inherit everything from cosmopolitan.meta except description
- remove meta.homepage.